### PR TITLE
ERP-1535 Proper tab-handling, proper handling of selection when user …

### PR DESCRIPTION
**Pull request by sdorheim**
…changes the input-date. Allow empty min and max value. Added possibility to have empty values

### CODE REVIEW

File: actindo-date-picker.html [hunk 1]
```diff
@@ -278,6 +294,11 @@
                 return !this.invalid;
             }
 
+            _isValidDate(date) {
+                return date && date.getTime && !isNaN(date.getTime());
```
Comment (cdreisbach) [inline actindo-date-picker.html]: date.getTime() - brackets missing

File: actindo-date-picker.html [hunk 2]
```diff
@@ -258,11 +269,16 @@
              */
             validate() {
                 let me = this;
+                if (me.allowEmpty &&
+                    (!me.value ||
+                        (me.value.valueOf() === 'string') && me.value.length === 0)) {
```
Comment (dev756) [inline actindo-date-picker.html]: closing bracket at wrong position

Comment (dev756) [reply actindo-date-picker.html]: .valueOf wrong use typeof

**Review by dev756 — CHANGES_REQUESTED** [review]

**Review by dev756 — APPROVED** [review]

File: actindo-date-picker.html [hunk 3]
```diff
@@ -270,8 +270,11 @@
             validate() {
                 let me = this;
                 if (me.allowEmpty &&
-                    (!me.value ||
-                        (me.value.valueOf() === 'string') && me.value.length === 0)) {
+                    (
+                        !me.value ||
+                        (me.value.typeOf() === 'string' && me.value.length === 0)
```
Comment (dev756) [inline actindo-date-picker.html]: typeof me.value

**Review by dev756 — CHANGES_REQUESTED** [review]

**Pull request merged**